pages/Documents/ConversationalAI/Third-PartyBots/e-bot7/introduction.md

```diff
@@ -0,0 +1,49 @@
+---
+pagename: Introduction
+redirect_from:
+  - bot-connectors-ebot7.html
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: e-bot7
+permalink: third-party-bots-e-bot7-introduction.html
+indicator:
+---
+
+### Overview
+
+{: .notice}
+Please note the e-bot7 connector is only accessible for early adopters. Enabling the use requires a whitelisting done by
+LivePerson. Contact your LivePerson account representative for assistance.
+
+The following document outlines the configuration of the **e-bot7** connector.
+
+The e-bot7 feature of Third Party Bots allows brands to connect to their existing bots in the ecosystem of e-bot7
+as a service of bot responses inside the Conversational Cloud.
+
+{: .important}
+To use this integration you need to make sure to have set up an account with e-bot7, have an existing bot, and created an application to be able to connect with third-party systems. You can read more about applications in the [documentation of e-bot7](https://docs.e-bot7.com/docs/applications/how_apps_work).
+
+### Configuration
+
+{: .important}
+See the [Getting Started](third-party-bots-getting-started.html) guide before using this document to complete pre-requisite steps.
+
+You will be presented with the following screen to complete the Vendor Settings in order to add bot connector.
+
+<img class="fancyimage" style="width:600px" src="img/ThirdPartyBots/e-bot7-configuration.png">
+
+Figure 1.1 e-bot7 Vendor Step
```

Comment:
Please use `<figure>` markup as in other cases. (Pending other feedback around the use of HTML markup itself.)

Reply:
@j9t Added `<figure>` here